WordPress/src/main/java/org/wordpress/android/workers/reminder/ReminderScheduler.kt

```diff
@@ -16,10 +16,12 @@ class ReminderScheduler @Inject constructor(
     val contextProvider: ContextProvider
 ) {
     val workManager by lazy { WorkManager.getInstance(contextProvider.getContext()) }
+    var hour: Int = DEFAUlT_START_HOUR
+    var minute: Int = DEFAULT_START_MINUTE
```

Comment:
Instead of adding new mutable variables here, we should probably either add a `time` argument to the `schedule` function or refactor `ReminderConfig` to handle `LocalDateTime`s instead of `LocalDate`s. The first option seems simpler.